src/renderer/base/thread.hpp

```diff
@@ -37,8 +37,6 @@ namespace Microsoft::Console::Render
         static DWORD WINAPI s_ThreadProc(_In_ LPVOID lpParameter);
         DWORD WINAPI _ThreadProc();
 
-        static DWORD const s_FrameLimitMilliseconds = 8;

```

Comment:
I don't actually know why it's 8 and not 6 or 10... 😰
Do you mean I should add comments saying why we Sleep()? Because I don't think I can comment why it's 8 specifically...

Reply:
Nah, I was more curious about the `8` in particular. But if we don't know where it comes from, it's fine then.